api.go

```diff
@@ -55,6 +55,7 @@ func createPodFromConfig(podConfig PodConfig) (*Pod, error) {
 	}
 
 	// Initialize the network.
+	DefaultNetInterworkingModel = p.config.NetworkConfig.DefaultNetInterworkingModel
```

Comment:
A few questions:

- Where is `p.config.NetworkConfig.DefaultNetInterworkingMode` being set?
- Doesn't this mean that the initial value of `DefaultNetInterworkingModel` (`ModelMacVtap`) isn't used now?
- If we need to retain the initial `ModelMacVtap`, maybe you could add a log call along the lines of:
  ```
  newNetModel := p.config.NetworkConfig.DefaultNetInterworkingModel

  if newNetModel != DefaultNetInterworkingModel {
      virtLog.WithFields(logrus.Fields{
          "old-network-model": DefaultNetInterworkingModel,
          "new-network-model": newNetworkModel,
      }).Info("switching network model")

      DefaultNetInterworkingModel = newNetModel
  }
  ```

Reply:
- p.config.NetworkConfig.DefaultNetInterworkingMode,  I am setting from the runtime. I mean the configuration provided by the caller to api.CreatePod should set it.
- Yes, DefaultNetInterworkingModel is being used now, but the api user (the runtime) should be able to modify what network interface need to be used ( as we support differents today). 
- Agree to log what is the default networking model. I will add it